File: AlgoMethod/fullSearch/1_array/7.cpp
//2022/08/13 17:02:54
//2022/08/13 17:04:36 AC.
#include <iostream>
#include <vector>
using namespace std;
int main(){
    int N;
    cin >> N;
    vector<int> A(N);
    for (int i = 0; i < N; i++) {
        cin >> A[i];
    }
    int cnt = 0, max = 0;
    for (int i = 0; i < N; i++) {
        if (max < A[i]) {
            max = A[i];
            cnt = i;
        }
    }
    cout << cnt << endl;
    return 0;
}

File: AlgoMethod/Python/5/7.py
# 2023/06/11 09:08:55
# 2023/06/11 09:10:44 AC.
X = 27
cnt = 0
while X != 1:
    if X % 2 == 1:
        X = X * 3 + 1
    else:
        X //= 2
    cnt += 1
print(cnt)
